README.md

```diff
@@ -1,6 +1,10 @@
-<a href="https://bokeh.org">
-  <img src="https://static.bokeh.org/logos/logotype.svg" height="60" alt="Bokeh logotype" />
-</a>
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/bokeh/pm/main/assets/logos/SVG/bokeh-logo-black-text-no-padding.svg">
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/bokeh/pm/main/assets/logos/SVG/bokeh-logo-white-text-no-padding.svg">
+    <img alt="Bokeh logo -- text is white in dark theme and black in light theme" width="30%"/>
+  </picture>
+</p>
```

Comment:
```suggestion
<picture>
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/bokeh/pm/main/assets/logos/SVG/bokeh-logo-black-text-no-padding.svg">
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/bokeh/pm/main/assets/logos/SVG/bokeh-logo-white-text-no-padding.svg">
  <img alt="Bokeh logo -- text is white in dark theme and black in light theme" height=60/>
</picture>
```

Reply:
LGTM!

Reply:
I think you should keep the `src="https://whatever-the-black-text-logo-is.svg"` in the `<img ...>` tag for backward compatibility. Otherwise old browsers that don't understand the `prefers-color-scheme` media query won't display any logo at all.

Then, strictly speaking, you don't need the `prefers-color-scheme: light` line as it is duplicating the same image that is in the `<img src=...>` tag.

Reply:
Shall do! Thank you, Ian!